src/lib.rs

```diff
@@ -481,6 +481,43 @@ impl<T: Debug + PrimInt + One + Zero> Vob<T> {
         }
     }
 
+    ///Returns contents of Vob as Vec of bytes.
```

Comment:
All other coments in this file are `/// XXX` i.e. a space after `///`. Also, the first sentence currently doesn't parse.

You also need to run `cargo fmt` over this in order to shut Travis up.

Reply:
Yes

Reply:
It sounds like you're using an old version of rustfmt. Personally I use the nightly version.

Reply:
Thanks, I used ```cargo +nightly install --force rustfmt-nightly``` and then ran ```cargo +nightly fmt``` and now it works fine